Schema combatibility issues for Altoida data?

My understanding is our schema-registry set to compatibility :NONE 
But I am still experiencing on the server where RADAR-Schemas:0.5.6 is deployed.

```[2020-01-24 17:38:26,931] ERROR WorkerSourceTask{id=radar-upload-source-2} Task threw an uncaught and unrecoverable exception (org.apache.kafka.connect.runtime.WorkerTask)
org.apache.kafka.connect.errors.ConnectException: Tolerance exceeded in error handler
	at org.apache.kafka.connect.runtime.errors.RetryWithToleranceOperator.execAndHandleError(RetryWithToleranceOperator.java:178)
	at org.apache.kafka.connect.runtime.errors.RetryWithToleranceOperator.execute(RetryWithToleranceOperator.java:104)
	at org.apache.kafka.connect.runtime.WorkerSourceTask.convertTransformedRecord(WorkerSourceTask.java:270)
	at org.apache.kafka.connect.runtime.WorkerSourceTask.sendRecords(WorkerSourceTask.java:294)
	at org.apache.kafka.connect.runtime.WorkerSourceTask.execute(WorkerSourceTask.java:229)
	at org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:177)
	at org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:227)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.kafka.connect.errors.DataException: Failed to serialize Avro data from topic connect_upload_altoida_tap :
	at io.confluent.connect.avro.AvroConverter.fromConnectData(AvroConverter.java:83)
	at org.apache.kafka.connect.runtime.WorkerSourceTask.lambda$convertTransformedRecord$2(WorkerSourceTask.java:270)
	at org.apache.kafka.connect.runtime.errors.RetryWithToleranceOperator.execAndRetry(RetryWithToleranceOperator.java:128)
	at org.apache.kafka.connect.runtime.errors.RetryWithToleranceOperator.execAndHandleError(RetryWithToleranceOperator.java:162)
	... 11 more
Caused by: org.apache.kafka.common.errors.SerializationException: Error registering Avro schema: {"type":"record","name":"AltoidaTouch","namespace":"org.radarcns.connector.upload.altoida","fields":[{"name":"time","type":{"type":"double","connect.doc":"Timestamp in unix time acquired locally from the device."},"doc":"Timestamp in unix time acquired locally from the device."},{"name":"timeReceived","type":{"type":"double","connect.doc":"Timestamp in unix time received from Altoida API."},"doc":"Timestamp in unix time received from Altoida API."},{"name":"x","type":{"type":"double","connect.doc":"Distance from the center of the screen in (cm)s on x-axis."},"doc":"Distance from the center of the screen in (cm)s on x-axis."},{"name":"y","type":{"type":"double","connect.doc":"Distance from the center of the screen in (cm)s on y-axis."},"doc":"Distance from the center of the screen in (cm)s on y-axis."},{"name":"surface","type":{"type":"double","connect.doc":"Diameter of the touch area in cm."},"doc":"Diameter of the touch area in cm."},{"name":"accelerometer","type":{"type":"double","connect.doc":"Acceleration exerted by the finger hitting the screen."},"doc":"Acceleration exerted by the finger hitting the screen."},{"name":"combined","type":{"type":"double","connect.doc":"An overall value of touch pressure formed from a linear combination of the previous columns."},"doc":"An overall value of touch pressure formed from a linear combination of the previous columns."},{"name":"hit","type":{"type":"boolean","connect.doc":"Whether the touch hit a button or not, 1 if the touch served a purpose basically, 0 otherwise."},"doc":"Whether the touch hit a button or not, 1 if the touch served a purpose basically, 0 otherwise."}],"connect.doc":"Location where the user touched the touchscreen on an x and y axis. These values are calculated from the center of the screen in centimeters (cm).","connect.name":"org.radarcns.connector.upload.altoida.AltoidaTouch"}
Caused by: io.confluent.kafka.schemaregistry.client.rest.exceptions.RestClientException: Schema being registered is incompatible with an earlier schema; error code: 409
	at io.confluent.kafka.schemaregistry.client.rest.RestService.sendHttpRequest(RestService.java:230)
	at io.confluent.kafka.schemaregistry.client.rest.RestService.httpRequest(RestService.java:256)
	at io.confluent.kafka.schemaregistry.client.rest.RestService.registerSchema(RestService.java:356)
	at io.confluent.kafka.schemaregistry.client.rest.RestService.registerSchema(RestService.java:348)
	at io.confluent.kafka.schemaregistry.client.rest.RestService.registerSchema(RestService.java:334)
	at io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient.registerAndGetId(CachedSchemaRegistryClient.java:168)
	at io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient.register(CachedSchemaRegistryClient.java:222)
	at io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient.register(CachedSchemaRegistryClient.java:198)
	at io.confluent.kafka.serializers.AbstractKafkaAvroSerializer.serializeImpl(AbstractKafkaAvroSerializer.java:70)
	at io.confluent.connect.avro.AvroConverter$Serializer.serialize(AvroConverter.java:131)
	at io.confluent.connect.avro.AvroConverter.fromConnectData(AvroConverter.java:80)
	at org.apache.kafka.connect.runtime.WorkerSourceTask.lambda$convertTransformedRecord$2(WorkerSourceTask.java:270)
	at org.apache.kafka.connect.runtime.errors.RetryWithToleranceOperator.execAndRetry(RetryWithToleranceOperator.java:128)
	at org.apache.kafka.connect.runtime.errors.RetryWithToleranceOperator.execAndHandleError(RetryWithToleranceOperator.java:162)
	at org.apache.kafka.connect.runtime.errors.RetryWithToleranceOperator.execute(RetryWithToleranceOperator.java:104)
	at org.apache.kafka.connect.runtime.WorkerSourceTask.convertTransformedRecord(WorkerSourceTask.java:270)
	at org.apache.kafka.connect.runtime.WorkerSourceTask.sendRecords(WorkerSourceTask.java:294)
	at org.apache.kafka.connect.runtime.WorkerSourceTask.execute(WorkerSourceTask.java:229)
	at org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:177)
	at org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:227)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

--- a/kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/altoida/AltoidaTouchScreenCsvProcessor.kt
+++ b/kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/altoida/AltoidaTouchScreenCsvProcessor.kt
@@ -28,7 +28,7 @@ class AltoidaTouchScreenCsvProcessor : StatelessCsvLineProcessor() {
     override fun lineConversion(
             line: Map<String, String>,
             timeReceived: Double
-    ) = TopicData("connect_upload_altoida_tap", AltoidaTouch(
+    ) = TopicData("connect_upload_altoida_touch", AltoidaTouch(
             time(line),
             timeReceived,
             line.getValue("X").toDouble(),

